scopes/preview/preview/pre-bundle.task.ts

```diff
@@ -0,0 +1,52 @@
+import { join } from 'path';
+import { BuildContext, BuildTask, BuiltTaskResult, TaskLocation } from '@teambit/builder';
+import { Capsule } from '@teambit/isolator';
+import { Logger } from '@teambit/logger';
+import { UIRoot, UiMain } from '@teambit/ui';
+import { generateBundleHash, getBundleArtifactDef } from './pre-bundle-utils';
+import { RUNTIME_NAME, buildPreBundlePreview } from './pre-bundle';
+
+export const BUNDLE_TASK_NAME = 'PreBundlePreview';
+export const BUNDLE_DIR = 'ui-bundle';
+
+export class PreBundlePreviewTask implements BuildTask {
+  aspectId = 'teambit.preview/preview';
+  name = BUNDLE_TASK_NAME;
+  location: TaskLocation = 'end';
+
+  constructor(private ui: UiMain, private logger: Logger) {}
+
+  async execute(context: BuildContext): Promise<BuiltTaskResult> {
+    const capsule: Capsule | undefined = context.capsuleNetwork.seedersCapsules.find(
+      (c) => c.component.id.toStringWithoutVersion() === 'teambit.preview/preview'
+    );
+    if (!capsule) {
+      return { componentsResults: [] };
+    }
+
+    try {
+      const outputPath = join(capsule.path, 'artifacts', BUNDLE_DIR);
```

Comment:
take the word artefacts from const, search for something like ARTIFACTS_DIR or something like this. I think it's in the builder aspect